Makefile

```diff
@@ -56,10 +59,10 @@ ifeq ($(ORIG_COMPILER),1)
   CC_OLD    = $(QEMU_IRIX) -L tools/ido5.3_compiler tools/ido5.3_compiler/usr/bin/cc
 endif
 
-AS         := $(MIPS_BINUTILS_PREFIX)as
-LD         := $(MIPS_BINUTILS_PREFIX)ld
-OBJCOPY    := $(MIPS_BINUTILS_PREFIX)objcopy
-OBJDUMP    := $(MIPS_BINUTILS_PREFIX)objdump
+AS         := $(MIPS_BINUTILS_PREFIX)-as
+LD         := $(MIPS_BINUTILS_PREFIX)-ld
+OBJCOPY    := $(MIPS_BINUTILS_PREFIX)-objcopy
+OBJDUMP    := $(MIPS_BINUTILS_PREFIX)-objdump
```

Comment:
there's no reason to move the `-` from `MIPS_BINUTILS_PREFIX` to every single usage.
It actually breaks compiling on native mips linux systems where the prefix would be empty.